src/react/index.ts

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {provideReactWrapper} from '@microsoft/fast-react-wrapper';
+import {provideFASTDesignSystem} from '@microsoft/fast-components';
+
+import {
+	vsCodeButton,
+	vsCodeCheckbox,
+	vsCodeBadge,
+	vsCodeDivider,
+	vsCodeLink,
+	vsCodeProgressRing,
+	vsCodeTag,
+	vsCodeTextArea,
+	vsCodeTextField,
+	vsCodeOption,
+	vsCodeDropdown,
+	vsCodePanels,
+	vsCodePanelTab,
+	vsCodePanelView,
+	vsCodeRadio,
+	vsCodeRadioGroup,
+	vsCodeDataGrid,
+	vsCodeDataGridCell,
+	vsCodeDataGridRow,
+} from '..';
```

Comment:
We have an eslint config that enforces alphabetical imports which is why you're getting a build error here.

You can use `npm run lint:fix` to automatically fix this error locally then push up the changes.

Reply:
Also have `npm run fmt:fix` for any formatting errors you may run into as an FYI.

Reply:
resolved by 15c25102230abfd0e64c48c89c8d5d98b070c6c1